float32[2] center_pixels
float32[2] center_inertial
float32[2] sonar_pos_pixels